include/fuse_kernel.h

```diff
@@ -104,8 +104,8 @@
  *  - add FUSE_NO_OPEN_SUPPORT flag
  */
 
-#ifndef _LINUX_FUSE_H
```

Comment:
I think this file should not be changed unless you also get the changes into the kernel (http://lxr.free-electrons.com/source/include/uapi/linux/fuse.h#L111). Other than that, this looks fine.